pkg/datarepair/checker/checker_test.go

```diff
@@ -200,7 +209,11 @@ func BenchmarkIdentifyInjuredSegments(b *testing.B) {
 	b.ResetTimer()
 	for i := 0; i < b.N; i++ {
 		interval := time.Second
-		checker := newChecker(pointerdb, repairQueue, overlayServer, limit, logger, interval)
+		irrdb, err := irreparabledb.New("sqlite3://file::memory:?mode=memory&cache=shared")
+		defer irrdb.Close()
```

Comment:
Don't use defer inside a loop. It won't close until it returns from loop. https://blog.learngoprogramming.com/gotchas-of-defer-in-go-1-8d070894cb01